frontend/src/concepts/pipelines/content/compareRuns/metricsSection/types.ts

```diff
@@ -0,0 +1,19 @@
+// An artifact which has associated event.
+
+import { LinkedArtifact } from '~/concepts/pipelines/apiHooks/mlmd/types';
+import { PipelineRunKFv2 } from '~/concepts/pipelines/kfTypes';
+import { Artifact, Event, Execution } from '~/third_party/mlmd';
+
+// each run can have multiple executions, artifacts, and events
+export type MlmdPackage = {
```

Comment:
nit: is "Package" the best way to describe what this object represents? If all of the data relates to the run, should the name not indicate this in some way?

Reply:
its what KF used, i can come up with a better one